src/vs/code/electron-sandbox/workbench/workbench-dev.html

```diff
@@ -56,6 +56,7 @@
 					domLineBreaksComputer
 					dompurify
 					editorGhostText
+					editorInlineEdit
```

Comment:
Try to not touch this list. github.dev etc. has to be updated for this as well. Cannot you reuse `editorGhostText`?

Reply:
✅